sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -133,6 +134,9 @@ def __init__(
         except AttributeError:
             raise ValueError("Blob URL must be a string.")
         parsed_url = urlparse(blob_url.rstrip('/'))
+
+        self.protocol = parsed_url.scheme
```

Comment:
Please remove this - this value is already stored in `self.scheme`
https://github.com/Azure/azure-sdk-for-python/blob/feature/storage-preview3/sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py#L70